public/app/core/services/backend_srv.ts

```diff
@@ -433,8 +433,13 @@ export class BackendSrv implements BackendService {
     return this.get<DashboardDTO>(`/api/public/dashboards/${uid}`);
   }
 
-  getFolderByUid(uid: string) {
-    return this.get<FolderDTO>(`/api/folders/${uid}`);
+  getFolderByUid(uid: string, options: { withAccessControl?: boolean } = {}) {
```

Comment:
nit: not a big fan of these inline type definitions, can we move this to a separate interface?